src/styles/normalize.css

```diff
@@ -0,0 +1,4 @@
+* {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
```

Comment:
1 - use `html` tag for that purpose
2 - font-weight: 400 - is a default value, can be removed